src/azure-cli/azure/cli/command_modules/vm/tests/latest/test_vm_commands.py

```diff
@@ -865,6 +865,42 @@ def test_vm_create_state_modifications(self, resource_group):
         self.cmd('vm list --resource-group {rg}', checks=self.is_empty())
 
 
+class VMSimulateEvictionScenarioTest(ScenarioTest):
+
+    @ResourceGroupPreparer(name_prefix='cli_test_vm_simulate_eviction')
+    def test_vm_simulate_eviction(self, resource_group):
+
+        self.kwargs.update({
+            'loc': 'eastus',
+            'vm1': 'vm-simualte-eviction1',
+            'vm2': 'vm-simulate-eviction2',
+            'vm3': 'vm-simulate-eviction3'
+        })
+
+        # simulate-eviction on a Regular VM, expect failure
+        self.cmd('vm create --resource-group {rg} --name {vm1} --location {loc} --image Centos --priority Regular')
+        self.cmd('vm simulate-eviction --resource-group {rg} --name {vm1}', expect_failure=True)
+
+        # simulate-eviction on a Spot VM with Deallocate policy, expect VM to be deallocated
+        self.cmd('vm create --resource-group {rg} --name {vm2} --location {loc} --image Centos --priority Spot --eviction-policy Deallocate')
```

Comment:
Add `--admin-username` in `vm/vmss create` test before #14265 is fixed.

Reply:
As we disussed, if this can not be fixed in your side, anthor pr will be opened to fix it.